src/search_file.c

```diff
@@ -166,7 +166,10 @@ static long get_bundle_size(const char *bundle)
 		return b->size;
 	}
 
-	b = calloc(1, sizeof(struct bundle_size));
+	b = (struct bundle_size*)calloc(1, sizeof(struct bundle_size));
+	if (!b) {
+		abort();
```

Comment:
oh, It's better not to call abort here. Use the ON_NULL_ABORT macro

Reply:
ahh good call